lowlevel_name: 'BlueSlam'
drive_rate: 50
vel_axis: 1
omega_axis: 0
enable_button: 3
vel_ramp: 1.0
omega_ramp: 1.0
